asset/css/variables.less

```diff
@@ -162,5 +181,13 @@
     --suggestions-relation-path-focus-bg: var(--base-gray);
 
     --card-border-color: var(--base-gray-light);
+
+    --schedule-element-fields-bg: var(--default-input-bg);
+    --schedule-element-default-text-color: var(--base-primary-color);
+
+    --schedule-element-fields-hover-bg: @base-primary-light;
```

Comment:
In light mode, using a less variable is the same as an absolute value in dark mode. Please avoid this on specific variables.

Reply:
💡✅